src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-04-22
+modDate: 2024-04-22
+title: Kubernetes agent
+description: How to configure a Kubernetes agent as a deployment target in Octopus
+navOrder: 10
+---
+
+Kubernetes agent targets are a mechanism for executing [Kubernetes steps](/docs/deployments/kubernetes) from inside the target Kubernetes cluster, rather than via an external API connection.
+
+Similar to the [Octopus Tentacle](/docs/infrastructure/deployment-targets/tentacle), the Kubernetes agent is a small, lightweight application that is installed into the target Kubernetes cluster.
+
+The Kubernetes agent provides a number of improvements over the [Kubernetes API](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-api) target:
+
+### Polling communication
+
+The agent uses the same [polling communication](/docs/infrastructure/deployment-targets/tentacle/tentacle-communication#polling-tentacles) protocol as [Octopus Tentacle](/docs/infrastructure/deployment-targets/tentacle). It lets the agent initiate the connection from the cluster to Octopus Server, solving network access issues such as publicly addressable clusters..
+
+### In-cluster authentication
+
+As the agent is already running inside the target cluster, Octopus Server no longer needs authentication credentials to the cluster to perform deployments. It can use the in-cluster authentication support of Kubernetes to run deployments using Kubernetes Service Accounts and Kubernetes RBAC local to the cluster.
+
+### Cluster-aware tooling
+
+As the agent is running in the cluster, it can retrieve the cluster's version and correctly use tooling that's specific to that version. You also need a lot less tooling as there are no longer any requirements for custom authentication plugins.
+
+## Requirements
+
+The Kubernetes agent is supported on the following versions:
+* Octopus Server **2024.2.6580** or newer
+* Kubernetes **1.26** to **1.29** (inclusive)
+
+Additionally, the Kubernetes agent only supports **Linux AMD64** and **Linux ARM64** Kubernetes nodes.
+
+## Installing the Kubernetes agent
+
+The Kubernetes agent is installed using [Helm](https://helm.sh) via the [octopusdeploy/kubernetes-agent](https://hub.docker.com/r/octopusdeploy/kubernetes-agent).
```

Comment:
via the `octopusdeploy/kubernetes-agent` "chart"?

Reply:
fixed